stout/flags.cc

```diff
@@ -99,8 +238,59 @@ void Parser::Parse(int* argc, const char*** argv) {
       break;
     }
 
-    // Skip anything that doesn't look like a flag.
+    // Skip anything that doesn't look like a flag, subcommand or positional
+    // argument.
     if (arg.find("--") != 0) {
+      // It could be probably subcommand.
+      const auto nested_parser_iter = std::find_if(
```

Comment:
exit if not found first please

Reply:
Nope, if we do not find any nested parser for this arg => it might be a positional argument.